cpu/lpc11u34/include/periph_cpu.h

```diff
@@ -62,6 +62,11 @@ typedef uint16_t gpio_t;
 #define GPIO_PIN(port, pin)     (gpio_t)((port << 16) | pin)
 
 /**
+ * @brief   Number of PWM channels per PWM peripheral
+ */
+#define PWM_CHAN_NUMOF      (3U)
```

Comment:
What I thought was, what if the board just defines two PWMs. Does it need to be `3` for the CPU code